src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -847,6 +847,30 @@ def snapshot_info(self, snapname):
                                       "snapshot '{0}' does not exist".format(snapname))
             raise VolumeException(-e.args[0], e.args[1])
 
+    def check_dangling_clone_index(self, snapname):
+        try:
+            if self.has_pending_clones(snapname):
+                pending_track_id_list = self.metadata_mgr.list_all_keys_with_specified_values_from_section('clone snaps', snapname)
```

Comment:
such a long name... `list_all_keys` should've sufficed 😅

Reply:
It makes it more descriptive, but I am on the fence here. @kotreshhr

Reply:
I'm for a descriptive function name! 
This one specifically could be shortened to `keys_with_value_from_section`.
Another option in python would be using kwargs:

``` mgr.list_keys(section='section', value='value') ```

Reply:
just thought that `filter_keys(**kwargs)` could be an even better alternative

Reply:
> just thought that `filter_keys(**kwargs)` could be an even better alternative

I did update!

Reply:
Thanks, resolving this comment.